library/public/camera.h

```diff
@@ -0,0 +1,90 @@
+/**
+ * @class   camera
+ * @brief   Abstract class to control a camera in a window
+ *
+ * A class to control a camera in a window.
+ * There is in essence three API.
+ * The Position/FocalPoint/ViewUp API, the ViewMatrix API and the camera movements API.
```

Comment:
Maybe add a comment that `SetViewUp` orthogonalize the input, if it's not orthogonal to the view direction.

Reply:
almost all methods OrthogonalizeViewUp. Should I mention which ones ?

Reply:
Ah ok. Then, maybe just add a note that the up direction is orthogonalized automatically if it's not orthogonal to the view direction defined by the position and the focal point.